src/server/graphql/models/CachedUser/cachedUserMutation.js

```diff
@@ -1,11 +1,11 @@
-import r from '../../../database/rethinkdriver';
+import r from '../../../database/rethinkdriver'; // eslint-disable-line id-length
```

Comment:
Ok. I agreed. @jrwells and I had a discussion around this, let's allow id-length of 1. @jrwells mind updating the rule set?